HistoryService/client/components/institution.jsx

```diff
@@ -117,104 +101,18 @@ class Institution extends React.Component {
-                      }}
-                    ></p>
-                  </li>
-
-                  <li className="list-group-item">
-                    <p id="FuturaStdHeavy">Type</p>
-                    <p
-                      id="Baskerville"
-                      dangerouslySetInnerHTML={{
-                        __html: this.state.article[0].artwork_type
-                      }}
-                    ></p>
-                  </li>
-
-                  <li className="list-group-item">
-                    <p id="FuturaStdHeavy">Culture Group</p>
-                    <p
-                      id="Baskerville"
-                      dangerouslySetInnerHTML={{
-                        __html: this.state.article[0].culture_group
-                      }}
-                    ></p>
-                  </li>
-
-                  <li className="list-group-item">
-                    <p id="FuturaStdHeavy">Material</p>
-                    <p
-                      id="Baskerville"
-                      dangerouslySetInnerHTML={{
-                        __html: this.state.article[0].material
-                      }}
-                    ></p>
-                  </li>
-
-                  <li className="list-group-item">
-                    <p id="FuturaStdHeavy">Tags</p>
-                    <p
-                      id="Baskerville"
-                      dangerouslySetInnerHTML={{
-                        __html: this.state.article[0].tags
-                      }}
-                    ></p>
-                  </li>
-
-                  <li className="list-group-item">
-                    <b>What Changed in last edit</b>
-                    {this.state.article[0].what_changed ? (
-                      <p id="Baskerville">
-                        {this.state.article[0].what_changed}
-                      </p>
-                    ) : (
-                      <p id="Baskerville">No information available</p>
-                    )}
-                  </li>
-                </ul>
-
-                {window.localStorage.getItem("admin") === "1" ? (
-                  <button
-                    className="btn btn-primary btn-block"
-                    onClick={this.deleteArticle}
-                  >
-                    Delete
-                  </button>
-                ) : (
-                  ""
-                )}
-              </div>
-            </div>
-
+            <InfoBox
+              photo={this.state.article[0].photo}
+              photo_license={this.state.article[0].photo_license}
+              user_name={user_name}
+              institution={this.state.article[0].institution}
+              artwork_type={this.state.article[0].artwork_type}
+              culture_group={this.state.article[0].culture_group}
+              material={this.state.article[0].material}
+              tags={this.state.article[0].tags}
+              what_changed={this.state.article[0].what_changed}
+              delete={this.deleteArticle}
```

Comment:
This code can be reduced if there's a check added to ensure `this.state.article[0]` exists before rendering (there's an example of this in `user_profile.jsx`).

To give an example here: `this.state.article[0]` can be set to a variable (like `article`) and that can reduce the veribosity of writing out each variable like `this.state.article[0].institution`, `this.state.article[0].artwork_type`, etc. The end result will look like `article.material`, `article.tags`, etc).

Reply:
Same thing with the `article.jsx` component, lines 121-130